prov/efa/src/rdm/rxr_ep.c

```diff
@@ -2189,6 +2190,8 @@ int rxr_endpoint(struct fid_domain *domain, struct fi_info *info,
 	if (ret)
 		goto err_free_ep;
 
+	rxr_peer_srx_construct(&rxr_ep->peer_srx);
```

Comment:
I can drop this change if requested. But I think this will not have any impact on the current code.